### PR TITLE
Louis/check

### PR DESCRIPTION
**Pull request by LinuxGit**
- refactor check role
- add disk fio read/write latency check
- add CPU, Mem, Disk space check
- add dev_mode

### CODE REVIEW

**Comment by LinuxGit** [comment]
@jomenxiao PTAL

File: roles/check_config_dynamic/tasks/main.yml [hunk 1]
```diff
@@ -1,36 +1,26 @@
 ---
 
-- name: disk space check - fail when disk is full
+- name: Disk space check - Fail task when disk is full
```
Comment (wyatt88) [inline roles/check_config_dynamic/tasks/main.yml]: This task is really needed in both check_config_dynamic and check_system_necessary roles?

Comment (LinuxGit) [reply roles/check_config_dynamic/tasks/main.yml]: start.yml/stop.yml and other playbooks will always include check_config_dynamic role.
Only bootstrap.yml include check_system_necessary role. it is said the task is a corner case.

**Comment by wyatt88** [comment]
LGTM

**Comment by jomenxiao** [comment]
LGTM

**Pull request merged**